spyder/widgets/findreplace.py

```diff
@@ -50,7 +50,14 @@ class FindReplace(QWidget):
     """Find widget"""
     STYLE = {False: "background-color:rgb(255, 175, 90);",
              True: "",
-             None: ""}
+             None: "",
+             'regexp_error': "background-color:rgb(255, 20, 20);",

```

Comment:
Could we add some alpha here to tone down a bit the this red? I mean, as it is, it's a bit too strong for the eye (in my opinion).

Reply:
Feel free do choose any shade of red you want. I didn't put any effort in chosing the color. IMO it should be a little stronger than the color for "not found".

There's no need for alpha though because it's the background anyway. Just adapt the rgb values.